### ISSUE TITLE
Suspend/Resume Cancels TBR, but isn't Reflected in HUD

### ISSUE THREAD

**Issue by shaun-22** [LoopKit/Loop#927]
**Describe the bug**
Suspending then resuming the pump while open looping cancels any current active TBR, but that change is not reflected in the app UI (HUD and Insulin Delivery section) which continue to display the previous TBR as active. 

**Attach an Issue Report**

https://loop.zulipchat.com/user_uploads/8196/CZDpFqV10kaY2_rSQJ9HsPUV/Loop-Report-2019-04-24-07_31_29-07_00.md

**To Reproduce**
1. Be open looping with an active TBR
2. Suspend then resume the pump.
3. Upon resume the pump will begin delivering scheduled basal rate, but the app continues to display the TBR as active.   

**Expected behavior**
The app interface will correctly reflect if a TBR is currently active or not. 

**Screenshots**
Pump is resumed at 7:22:43 AM and returns to scheduled basal rate:
![IMG_4358](https://user-images.githubusercontent.com/48635533/56686391-b9c82d80-6688-11e9-9f5d-21a63d2291db.PNG)

However, Loop UI continues to display active -1.05u/hour TBR:
![IMG_4351](https://user-images.githubusercontent.com/48635533/56685502-f561f800-6686-11e9-9863-dbd0159cea71.PNG)

**Phone**
 - Hardware: iPhone XS
 - Software: iOS 12.2

**Loop Version**
 - Version Number: 1.10.0DEV
 - Repo: LoopKit/Loop/omnipod-testing

**CGM**
 - Device: Dexcom G6
 - Manager app: Dexcom App

**Pump**
 - Manufacturer: Omnipod
 - Model: Eros

**Additional context**
I noticed the issue because I was hearing pulses from the pump despite the UI displaying the 0 TBR rate. Also, for what it's worth, Nightscout mirrors the bug in the app display; it incorrectly shows that the 0 temp continued. Not sure if that's relevant or not, but found it interesting. 
<img width="519" alt="Screen Shot 2019-04-24 at 11 53 53 AM" src="https://user-images.githubusercontent.com/48635533/56686454-debca080-6688-11e9-8323-eb652a60aade.png">

**Comment by shaun-22** [comment]
Github won't let me attach the Loop Report; I'm getting an unsupported file type error. Linking to version of the report hosted on Zulip from when I posted there for now.

**Comment by Nightfoxy** [comment]
When in closed loop, it would be great if Loop tried to restart the TBR after resume. In my case, my daughter was falling and a 0 TBR was set. I knew I needed to reduce the scheduled basal rates. I adjusted and "sync to pod", but that runs a suspend/resume, which canceled the 0 u/hr TBR and resumed the scheduled basal rate. This caused more insulin to be added to her falling BG and the insulin wasn't accounted for in the IOB math. I heard the pod clicking to verify.

@ps2 , does the fix you merged handle basal rate changes that trigger suspend/resume? Can closed loop trigger a re-evaluation of the basal rate so it could set a 0 u/hr rate right after updating basal rates in my case above?
